vcap/vcap/device_mapping.py

```diff
@@ -83,15 +84,49 @@ def filter_func(devices):
         return DeviceMapper(filter_func=filter_func)
 
     @staticmethod
-    def map_to_all_myriad(cpu_fallback=True):
-        """Returns Myriad device. If cpu_fallback=True, it will return CPU:0
-        if there are no myriad devices."""
+    def map_to_openvino_devices():
+        """Intelligently load capsules onto available OpenVINO compatible
+        devices.
+
+        Since support for OpenVINO devices is very experimental, there is a
+        temporary environment variable being added to whitelist devices
+        specifically. This variable will be deprecated and removed after a
+        short testing period.
+
+        OPENVINO_ALLOWABLE_DEVICES can be
+            "", "HDDL", or "MYRIAD"
+        The device "CPU" is _always_ allowed and always loaded onto.
+
+        The logic is as follows: IECore().available_devices is used to scan
```

Comment:
I appreciate the detailed documentation here, but I wonder if it could be simplified. Maybe something like:

In all cases, a backend will be mapped to the CPU device. The MYRIAD option will also map a backend to any available Myriad devices directly. The HDDL option will map a single backend to HDDL and let it schedule work for Myriad devices.

Reply:
I shortened it a bit!